### PR TITLE
Fix download link on some Linux platforms

### PR DESCRIPTION
**Pull request by WillowSauceR**
Screenshot_20221115-232752_Termux

### CODE REVIEW

**Comment by Torrekie** [comment]
need test on other platforms

**Comment by LNSSPsd** [comment]
I think installation on normal Linux distributions should be disabled by default for the possibility of messing up the environment.

**Comment by WillowSauceR** [comment]
> I think installation on normal Linux distributions should be disabled by default for the possibility of messing up the environment.

So what should we do? The current installation method for some Linux distributions is to copy fastbuilder to /usr/local/bin/fastbuilder and then add execute permissions to it.

**Comment by WillowSauceR** [comment]
> need test on other platforms

Testing on FreeBSD 13.1 x86_64, but it raises another issue, the file name in the link here is also wrong, it should be "phoenixbuilder-FreeBSD-executable-x86_64" instead of "phoenixbuilder-LFreeBSD- executable-x86_64"

image

**Comment by Torrekie** [comment]
it was a problem when using BSD `sed`, we need something to swap system name to lowercase but obviously that method we used was not suitable for BSD systems

File: install.sh [hunk 1]
```diff
@@ -273,7 +273,7 @@ elif [ ${MACHINE} == "macos" ]; then
 elif [[ ${SYSTEM_NAME} == "NetBSD" ]] || [[ ${SYSTEM_NAME} == "FreeBSD" ]] || [[ ${SYSTEM_NAME} == "OpenBSD" ]]; then
   echo           "If you met 404 error in further downloading, report it at"
   printf "\033[32m  https://github.com/LNSSPsd/PhoenixBuilder/issues\033[0m\n"
-  FB_PREFIX="phoenixbuilder-$(echo ${SYSTEM_NAME} | sed -e 's/\(.*\)/\L\1/')-executable-"
+  FB_PREFIX="phoenixbuilder-$(echo ${SYSTEM_NAME})-executable-"
```
Comment (Torrekie) [inline install.sh:276]:
```suggestion
  FB_PREFIX="phoenixbuilder-$(echo ${SYSTEM_NAME} | tr '[:upper:]' '[:lower:]')-executable-"
```
POSIX replacement

**Comment by WillowSauceR** [comment]
It worked: 
image

**Pull request merged**